src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -583,7 +583,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <Target Name="_GetReferenceAssemblyPaths">
 	<GetReferenceAssemblyPaths
 			TargetFrameworkMoniker="$(TargetFrameworkIdentifier),Version=v1.0"
-			RootPath="$(TargetFrameworkRootPath)">
+			RootPath="$(XATargetFrameworkRootPath)">
```

Comment:
...except this isn't entirely accurate: the [Linux PR #726 build failure](https://jenkins.mono-project.com/job/xamarin-anroid-linux-pr-builder/726/) is because we're *not* overriding `$(TargetFrameworkRootPath)`:

```
Target GetReferenceAssemblyPaths:
Task "CreateProperty"
        Using task CreateProperty from Microsoft.Build.Tasks.CreateProperty, Microsoft.Build.Tasks.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
Done executing task "CreateProperty"
Task "GetReferenceAssemblyPaths"
        Using task GetReferenceAssemblyPaths from Microsoft.Build.Tasks.GetReferenceAssemblyPaths, Microsoft.Build.Tasks.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
        Looking for framework 'MonoAndroid,Version=v7.1' in root path '/usr/lib/mono/4.5/../xbuild-frameworks'
        Unable to find framework definition file '/usr/lib/mono/4.5/../xbuild-frameworks/MonoAndroid/v7.1/RedistList/FrameworkList.xml' for Target Framework Moniker 'MonoAndroid,Version=v7.1'
Microsoft.Common.targets:  warning : Unable to find framework corresponding to the target framework moniker 'MonoAndroid,Version=v7.1'. Framework assembly references will be resolved from the GAC, which might not be the intended behavior.
Done executing task "GetReferenceAssemblyPaths"
Done building target "GetReferenceAssemblyPaths" in project "/mnt/jenkins/workspace/xamarin-anroid-linux-pr-builder/xamarin-android/src/Xamarin.Android.NUnitLite/Xamarin.Android.NUnitLite.csproj".
```

That raises an interesting conundrum: *it might not be possible* to **NOT** override `$(TargetFrameworkRootPath)`, which means the *only* way to "sanely" build on Windows is by "installing" it.

😢